SoraFoundation/Classes/NumberProcessing/TokenFormatter.swift

```diff
@@ -0,0 +1,52 @@
+/**
+* Copyright Soramitsu Co., Ltd. All Rights Reserved.
+* SPDX-License-Identifier: GPL-3.0
+*/
+
+
+import Foundation
+
+public enum TokenSymbolPosition {
+    case prefix
+    case suffix
+}
```

Comment:
Maybe it would be more convenient to use symbol as associated value and have something like
```
enum Unit {
    case prefix(String)
    case suffix(String)
}
```

Reply:
Firstly, I thought that it is a good idea then I thought about a use case when we have an image to display as a symbol. So, I decided to live it as it is for now. Let see how it goes when we use it